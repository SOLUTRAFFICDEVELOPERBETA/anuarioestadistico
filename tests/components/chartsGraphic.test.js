import '@testing-library/jest-dom';
import { mount, shallow } from 'enzyme';
import React from 'react';
import ChartGraphic from '../../components/chartsGraphic';
import { demoDataGraphic } from '../fixtures';

describe('Pruebas en <ChartGraphic/>', () => {
    const wrapper = mount(
        <ChartGraphic
            id={"chart-Prueba"}
            value={demoDataGraphic}

        />
    )
    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe existir el componente de default', ()=> {
        expect(wrapper.find('div').at(0).hasClass('MuiBox-root MuiBox-root-1')).toBe(true)
    })
})