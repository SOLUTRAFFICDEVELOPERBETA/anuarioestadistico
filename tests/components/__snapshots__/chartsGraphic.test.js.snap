// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <ChartGraphic/> debe mostrarse correctamente 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    style={
      Object {
        "width": "95%",
      }
    }
  />
</div>
`;
